tests/forcefields/test_jobs.py

```diff
@@ -66,6 +78,9 @@ def test_m3gnet_static_maker(si_structure):
 
 
 def test_m3gnet_relax_maker(si_structure):
+    # FIXME - brittle due to inability to adjust dtypes in M3GNetRelaxMaker
+    torch.set_default_dtype(torch.float32)
```

Comment:
Do we really need these? I think 

```py
        potential_kwargs={"default_dtype": "float64"},
```

should do it.

Reply:
Couldn't see where the dtypes can be set in chgnet codebase so need to do globally, issue is MACE does it globally to 64 so tests using chg/m3g after mace jobs test were failing

Reply:
I see. Pinging @ilyes319 and @davkovacs to see if this can be solved on MACE's side without changing global state?

For now, we'll wrap the test in `try/finally` and revert the `torch.dtype` at the end.

Reply:
> It would be great to make it safe to use different types of potential in one go.

Strongly agree. I think we can figure something out with Ilyes and David.

Reply:
Thanks!

Reply:
Yes dtype conversion has haunted me for a long time...